app/helpers/lookup_keys_helper.rb

```diff
@@ -118,7 +118,7 @@ def lookup_key_warnings(required, has_value)
       { :text => _("Required parameter without value.<br/><b>Please override!</b><br/>"),
         :icon => "error-circle-o" }
     else
-      { :text => _("Optional parameter without value.<br/><i>Will not be sent to Puppet.</i><br/>"),
+      { :text => _("Optional parameter without value.<br/><i>Still managed by Foreman, the value will be empty.</i><br/>"),
```

Comment:
In this case empty value will be sent:
![image](https://cloud.githubusercontent.com/assets/10046168/18079909/c7995520-6e9b-11e6-8ea7-2f32582cf609.png)

Will lead to this output in the enc:

```
  XO4txkfDEUD: ''
```